examples/meta/src/observers/gaussian_mixture_models.sg

```diff
@@ -0,0 +1,35 @@
+File f_feats_train = csv_file("../../data/classifier_4class_2d_linear_features_train.dat")
+
+Math:init_random(1)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+#![create_features]
+
+#![create_gmm_instance]
+int num_components = 3
+GMM gmm(num_components)
+#![create_gmm_instance]
+
+#![add_observer]
+ParameterObserver obs = parameter_observer("ParameterObserverLogger")
+gmm.subscribe(obs)
+#![add_observer]
+
+#![train_sample]
+gmm.put("features", features_train)
+gmm.train_em()
+#![train_sample]
+
+#![training_smem]
+obs.clear()
```

Comment:
Why throw away the old obs? Maybe just a new observer/model?

Reply:
It was just to also show that feature. Since it is training the GMM with two different methods, maybe the user is interested just into the final observations and therefore he will get rid of the previous ones :)

Reply:
I would always create a new instance to avoid state-changing confusions... we had done this before and it was causing trouble

Reply:
Okay! I will add a new observer then :)